include/boost/beast/_experimental/test/impl/stream.hpp

```diff
@@ -99,11 +98,10 @@ class stream::read_op : public stream::read_op_base
     void
     operator()(error_code ec) override
     {
-        net::post(
-            wg1_.get_executor(),
-            bind_handler(
-                std::move(fn_),
-                ec));
+
+        auto alloc = net::get_associated_allocator(fn_.h_);
+        wg1_.get_executor().post(
+            beast::bind_front_handler(std::move(fn_), ec), alloc);
```

Comment:
This doesn't really do anything, it all gets inlined anyway...

Reply:
There's a significant difference between this and a 2-arg post. The 2-arg post does something like this, not counting hooks and work guards:
```
auto alloc = net::get_associated_allocator(fn_.h_);
        wg1_.get_executor().post(
          [f = std::move(f)](){
              auto alloc = net::get_associated_allocator(f);
              net::get_associated_executor(f).dispatch(std::move(f));
          }, alloc);
```